<map id="armor_api.armor_exceptions.ArmorPyException" name="armor_api.armor_exceptions.ArmorPyException">
<area shape="rect" id="node1" title=" " alt="" coords="140,39,353,80"/>
<area shape="rect" id="node3" href="$classarmor__api_1_1armor__exceptions_1_1_armor_service_call_error.html" title=" " alt="" coords="401,5,615,47"/>
<area shape="rect" id="node4" href="$classarmor__api_1_1armor__exceptions_1_1_armor_service_internal_error.html" title=" " alt="" coords="401,71,615,112"/>
<area shape="rect" id="node2" title=" " alt="" coords="5,46,92,73"/>
</map>
